pandas/util/_test_decorators.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+
+def _safe_import(mod_name, min_version=None):
```

Comment:
this could replace ``pandas.util.testing.skip_if_no_package`` as well (or maybe be an input to it)

Reply:
since this is already in a private module, you can name it safe_import